### PR TITLE
Active observation toggle.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -88,6 +88,8 @@ lazy val graphql = project
   .in(file("common-graphql"))
   .dependsOn(model.jvm)
   .settings(commonSettings: _*)
+  .settings(commonJsLibSettings: _*)
+  .enablePlugins(ScalaJSPlugin)
```
Comment (rpiaggio) [inline build.sbt:92]: It seems this was necessary after all.

File: common/src/main/webapp/less/style.less [hunk 2]
```diff
@@ -1365,11 +1365,16 @@ html {
   justify-content: flex-end;
 }
 
-.obs-card-header {
+.obs-badge-header {
```
Comment (rpiaggio) [inline common/src/main/webapp/less/style.less:1368]: Unifying naming criteria.

File: common/src/main/webapp/less/style.less [hunk 3]
```diff
@@ -1688,3 +1693,11 @@ svg.padded-right-icon.svg-inline--fa,
     }
   }
 }
+
+.ui.toggle.checkbox.obs-active-status-toggle {
+  transform: rotate(270deg) scale(0.6) translate(0, 150%);
```
Comment (cquiroz) [inline common/src/main/webapp/less/style.less:1698]: 👍

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**